### DISCUSSION TITLE
Forestry Outfit Woodcutting Skill Calculator

### DISCUSSION THREAD

**Discussion by kturner638** [runelite/runelite#17497] [Ideas]
We need to add the Forestry Outfit to the Woodcutting Skill Calculator as a selectable option or update the name of the current "Lumberjack Outfit (2.5%)" to include the Forestry Outfit.
![WSK140224](https://github.com/runelite/runelite/assets/86120153/b3ea251a-8218-410c-8c5a-2feb89a3e3d8)

**Comment by Felanbird** [comment]
They give the same % boost you can just select the existing one

**Reply by kturner638** [reply]
Even though that is true. It ill help clarify the items that allow the bonus for those who do not know yet and those who have forgotten.
